extensions/applicationinsights-react-native/package.json

```diff
@@ -38,7 +38,7 @@
         "eslint-plugin-promise": "^5.1.0",
         "qunit": "^2.11.2",
         "react": "^17.0.2",
-        "react-native": "^0.67.4",
+        "react-native": "^0.68.0",
```

Comment:
Tried to work around the current Component Governance issue with React-Native -- but they need to release another version (0.68.0 was 6 days ago)